jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codemodernizer/CodeWhispererCodeModernizerSessionTest.kt

```diff
@@ -236,6 +236,45 @@ class CodeWhispererCodeModernizerSessionTest : CodeWhispererCodeModernizerTestBa
         }
     }
 
+    @Test
+    fun `CodeModernizerSession can create zip and replace Windows file path`() {
+        addFilesToProjectModule(
+            "src\\tmp.java",
+            "target\\smth.java",
+            "target\\somedir\\anotherthing.class",
+            "pom.xml",
+            "someModule\\pom.xml",
+            "someModule\\target\\smth.class",
+            "someModule\\src\\helloworld.java",
+        )
+        var file: File? = null
+        val rootManager = ModuleRootManager.getInstance(module)
+        val roots = rootManager.contentRoots
+        assertFalse(roots.isEmpty() || roots.size > 1)
```

Comment:
what exactly is the behavior if there are multiple content roots for a module?

Reply:
Good question, I don't know if it's possible for our case to have multiple roots, and the tests will just fail due to  `assertFalse(roots.isEmpty() || roots.size > 1)`